bundle/bin/rke2-uninstall.sh

```diff
@@ -109,9 +109,9 @@ uninstall_remove_files()
     fi
 
     log "Removing rke2 files"
-    $transactional_update find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name rke2-*.service -type f -delete
-    $transactional_update find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name rke2-*.env -type f -delete
-    find /etc/systemd/system -name rke2-*.service -type f -delete
+    $transactional_update find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name 'rke2-*.service' -delete
+    $transactional_update find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name 'rke2-*.env' -delete
+    find /etc/systemd/system -name rke2-*.service -delete
```

Comment:
Nits:
* consistently quote with double quotes, not single
* consistently quote find args, even when not using transactional-update

```suggestion
    $transactional_update find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name "rke2-*.service" -delete
    $transactional_update find "${INSTALL_RKE2_ROOT}/lib/systemd/system" -name "rke2-*.env" -delete
    find /etc/systemd/system -name "rke2-*.service" -delete
```